confluent_kafka/kafkatest/verifiable_consumer.py

```diff
@@ -131,13 +126,10 @@ def on_commit (self, err, partitions):
 
         self.send(d)
 
-
-    def do_commit (self, immediate=False, async=None):
+    def do_commit(self, immediate=False, async=None):
         """ Commit every 1000 messages or whenever there is a consume timeout
             or immediate. """
-        if (self.use_auto_commit or
-            self.consumed_msgs_at_last_commit + (0 if immediate else 1000) >
-            self.consumed_msgs):
+        if self.use_auto_commit or self.consumed_msgs_at_last_commit + (0 if immediate else 1000) > self.consumed_msgs:
```

Comment:
It actually just didn't like the `>` comparison being split across multiple lines, I've tweaked the style a bit. flake8 doesn't complain about it, but using parentheses when they aren't necessary isn't normal python style, but here it does avoid the `\` if the condition is split across multiple lines.